sdcm/nemesis.py

```diff
@@ -885,6 +885,9 @@ def add_ldap_configuration_to_node(node):
     @retrying(n=3, sleep_time=60, allowed_exceptions=(NodeSetupFailed, NodeSetupTimeout))
     def _add_and_init_new_cluster_node(self, old_node_ip=None, timeout=HOUR_IN_SEC * 6, rack=0):
         """When old_node_private_ip is not None replacement node procedure is initiated"""
+        # TODO: make it work when we have decommissioned (by nodetool) nodes.
```

Comment:
If it's k8s only comment, maybe caption it as such. (Or move it to the actual place needs to fixed)

Reply:
Done